package.json

```diff
@@ -18,9 +20,7 @@
     "core-js": "^2.6.5",
     "express": "^4.17.1",
     "register-service-worker": "^1.6.2",
-    "vue": "^2.6.10"
-  },
-  "devDependencies": {
```

Comment:
This is wrong,
We must keep separated dependencies and devDependencies.

Heroku will still install devDependencies which are necessary to build the app.